pkgs/applications/version-management/gitlab/gitaly/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, fetchFromGitLab, fetchFromGitHub, buildGoModule, ruby
 , bundlerEnv, pkg-config
-# libgit2 + dependencies
-, libgit2, openssl, zlib, pcre, http-parser }:
+# libgit2 (git2go only supports v1.1.x) + dependencies
+, libgit2_1_1, openssl, zlib, pcre, http-parser }:
```

Comment:
Can you do `gitaly = callPackage [...] { libgit2 = libgit2_1_1};` in all-packages.nix instead?
That way the override can be removed without even changing the expression.

Reply:
done :heavy_check_mark: